test/VoxaApp.spec.ts

```diff
@@ -312,6 +313,151 @@ describe("VoxaApp", () => {
     expect(statesDefinition.entry.called).to.be.true;
   });
 
+  it("should fulfill request", async () => {
+    const canFulfillIntent: canfulfill.CanFulfillIntent = {
+      canFulfill: "YES",
+      slots: {
+        slot1: {
+          canFulfill: "YES",
+          canUnderstand: "YES",
+        },
+      },
+    };
+
+    const voxaApp = new VoxaApp({ views, variables });
+    const alexaSkill = new AlexaPlatform(voxaApp);
+    voxaApp.onCanFulfillIntentRequest((alexaEvent: AlexaEvent, alexaReply: AlexaReply) => {
+      alexaReply.fulfillIntent("YES");
+      alexaReply.fulfillSlot("slot1", "YES", "YES");
+      return alexaReply;
+    });
+
+    const fulfillEvent = _.cloneDeep(event);
```

Comment:
we have a AlexaRequestBuilder to be used in tests, please update it to support this request type and let's use that instead

https://github.com/mediarain/voxa/blob/v3/test/tools.ts#L16